engines/ccgost/gost2001_keyx.c

```diff
@@ -42,7 +42,8 @@ static int VKO_compute_key(unsigned char *shared_key, size_t shared_key_size,
     Y = BN_CTX_get(ctx);
     EC_GROUP_get_order(EC_KEY_get0_group(priv_key), order, ctx);
     BN_mod_mul(p, key, UKM, order, ctx);
-    EC_POINT_mul(EC_KEY_get0_group(priv_key), pnt, NULL, pub_key, p, ctx);
+    if (!EC_POINT_mul(EC_KEY_get0_group(priv_key), pnt, NULL, pub_key, p, ctx))
+	goto err;
```

Comment:
Presumably if there's an error, then this function should return a value that indicates it? And its caller should test for it...

Is this the only function called that can return an error?